sr_logging_common/logging/check_log_size.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import shutil
+import rospy
+import logging_utils as utils
+
+
+if __name__ == '__main__':
+    rospy.init_node('check_log_size')
+
+    additional_required_disk_space = rospy.get_param('~additional_required_disk_space', 10) * utils.GIGABYTE
```

Comment:
Additional to what?

Reply:
See above comment

Reply:
https://github.com/shadow-robot/common_resources/pull/265/commits/b0f7deb1b0e4752bc730d5720b7b737e726eadd8